packages/core/src/messages/responses/topic-discontinuity.ts

```diff
@@ -28,6 +34,14 @@ export class TopicDiscontinuity {
     return this._newSequenceNumber;
   }
 
+  /**
+   * Returns the new sequence page after the discontinuity.
+   * @returns number
+   */
+  public newSequencePage(): number {
+    return this._newSequencePage;
+  }
+
   public toString(): string {
     const displayValue = `Last Sequence Number: ${this._lastSequenceNumber}; New Sequence Number: ${this._newSequenceNumber}`;
```

Comment:
We should also add the sequence page to the string representation. This can be really helpful for debugging.